_posts/develop-on-ipad.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "An iPad as a developer box. Pros and cons"
+author: Mauro Servienti
+synopsis: "Can we use a 10-inches iPad Air as a development device? Let's find out what was my recent."
+header_image: /img/posts/develop-on-ipad/header.jpg
+tags:
+- development
+---
+
+When my 5-year-old iPad Pro fell on the floor for the third time (not accidentally, one could argue when you have 5-year-old kids. Coincidence? I don't think so!), I decided it was time for a new iPad rather than spending a lot to fix it due to an expired warranty.
```

Comment:
```suggestion
When my 5-year-old iPad Pro fell on the floor for the third time (not accidentally, one could argue when you have a 5-year-old kid. Coincidence? I don't think so!), I decided it was time for a new iPad rather than spending a lot to fix it due to an expired warranty.
```